bpf/Makefile

```diff
@@ -87,7 +87,7 @@ MAX_BASE_OPTIONS = -DSKIP_DEBUG=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 \
 	-DMONITOR_AGGREGATION=3 -DCT_REPORT_FLAGS=0x0002 -DENABLE_HOST_FIREWALL=1
 ifeq ("$(KERNEL)","49")
 # IPSec is incompatible with BPF NodePort so we only enable on 4.9.
-MAX_BASE_OPTIONS += -DENABLE_IPSEC=1
+MAX_BASE_OPTIONS += -DENABLE_IPSEC=1 -DIP_POOLS=1
```

Comment:
Can `IP_POOLS` only be defined on 4.9?

Reply:
IP_POOLS is (or at least shortly will be) used on 4.14 kernels in EKS and 5.4 kernels in EKS. Also users can enable it almost anywhere by adding a subnet. There shouldn't be any kernel requirements around using it.

Reply:
There was a "bug" floating around where IP_POOLS without fib lookup helper would cause errors.

Reply:
Then it should be defined on line 87.

Reply:
My approach here wasn't too complicated, I just put `IP_POOLS` wherever IPSec is enabled. Given that it's directly tied into that feature, that makes more sense to me rather than splitting it up to somewhere else. If we are only defining ENABLE_IPSEC on 4.9 due to the nodeport incompatibility then it'll make no difference to define IP_POOLS above.

Unless you're saying that the BPF nodeport incompatibility is no longer the case and we can drop the 4.9 kernel check entirely?

Reply:
(sidenote, the naming of `IP_POOLS` sounds generic but it's really an IPSEC-specific config option so we could probably refactor that to make it clearer but I'm trying not to bit off too many changes at once ;) )

Reply:
also wait until we get backports sorted so we don't have to backport around naming.

Reply:
Got it, thanks for the explanation!